### ISSUE TITLE
yaml without FD_control

### ISSUE THREAD

**Issue by Arthanis58** [Jhsmit/PyHDX#353]
Hi, is it no longer possible to set backexchange by fixed percentage ? I tried based on #303 but it seems that no longer works after the data loading rework.

**Comment by Jhsmit** [comment]
Yes, at the moment its not possible, I've removed it since it made my life a bit easier and wasn't sure if it was being used. 
I can put it back in though. How are you using PyHDX? are you using the web interface? do you input your data through .yaml files or use the web interface?

Out of curiosity, are you using our web server or do you self-host PyHDX?

**Comment by Arthanis58** [comment]
I am using the web interface because I am very new to python, and I am hosting my own web server as the linked one didnt load for me. So far I have been uploading the files in the batch mode.

The fixed backexchange isnt critical for me, although I would still welcome the option if its not too much hassle for you. I was just wondering if I was doing something wrong.
